torch/fx/experimental/symbolic_shapes.py

```diff
@@ -824,6 +824,28 @@ def create_symbolic_sizes_strides_storage_offset(self, ex: torch.Tensor, source:
     def create_symintnode(self, sym: "sympy.Expr"):
         return SymInt(SymNode(sym, self, int))
 
+    def create_sym_number(self, val, source):
+        if isinstance(val, int):
+            return self.create_symint(val, source)
+        elif isinstance(val, float):
+            return self.create_symfloat(val, source)
+        else:
+            raise AssertionError(f'unrecognized type {type(val)}')
+
+    def create_symint(self, val, source):
+        # these symints are NOT duck typed, we don't expect them to
+        # necessarily have any relationship to sizes
+        symbol = Symbol(f"s{len(self.var_to_val)}", integer=True)
+        symbol.sources.append(source)
+        self.var_to_val[symbol] = sympy.Integer(val)
```

Comment:
If its already in val_to_var, should we return what's there and short circuit a little?